string.go

```diff
@@ -47,6 +48,17 @@ func (self *StringParser) New(profile *Profile, options *ordereddict.Dict) (Pars
 		result.options.Term = string(term)
 	}
 
+	// Add a termexpression if exist
+	termexpression, _ := options.GetString("term")
```

Comment:
Should this be "term_expr" ?

Reply:
your correct, seems like my mods had a slight error, this should be fixed now
![image](https://user-images.githubusercontent.com/13081800/126672573-4752fc18-42d3-4e5d-bb56-5a5e36ef32b4.png)